src/App.js

```diff
@@ -1,34 +1,25 @@
 import React from 'react';
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
+
 
-const todoList = [
-  {
-    id: 1,
-    title: "do the dishes",
-  },
-  {
-    id: 2,
-    title: "take out the trash",
-  },
-  {
-    id: 3,
-    title: "sweep the floor",
-  },
-];
 
 function App() {
+
+  // const [newTodo, setNewTodo] = React.useState('');
+  const [todoList, setTodoList] = React.useState([]);
+
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  };
+
   return (
     <div>
     <h1>Todo List</h1>
-      <ul>
-        {todoList.map(function(item){
-          return ( 
-            <li key ={item.id}>
-              {item.title}
-            </li>
-          );
-        })}
-      </ul>
-      </div>
+    <AddTodoForm onAddTodo={addTodo}/>
+    {/* <p>{newTodo}</p> */}
+    <TodoList todoList={todoList}/>
```

Comment:
formatting: same comment as before. you want to make sure to indent all child components/elements. for example here it should be: 
```
<div>
    <h1>Todo List</h1>
    <AddTodoForm onAddTodo={addTodo}/>
    <TodoList todoList={todoList}/>
</div>